docs/api-status.json

```diff
@@ -1761,6 +1761,36 @@
         "comment": "ModifySubuser - https://docs.ceph.com/en/latest/radosgw/adminops/#modify-subuser\n",
         "added_in_version": "v0.15.0",
         "expected_stable_version": "v0.17.0"
+      },
+      {
+        "name": "API.ListUsersBuckets",
+        "comment": "ListUsersBuckets will return the list of all users buckets without stat.\n",
+        "added_in_version": "v0.16.0",
```

Comment:
Sorry but this needs to be fixed, the v0.16 release has already occurred so this would need to be updated to v0.17.0.  I am aware that this is a pain and I plan on tweaking the process so contributors do not need to do this in the future. Unfortunately, for now you need to fix it.

Reply:
fixed version to v0.17.0

Reply:
> Sorry but this needs to be fixed, the v0.16 release has already occurred so this would need to be updated to v0.17.0. I am aware that this is a pain and I plan on tweaking the process so contributors do not need to do this in the future. Unfortunately, for now you need to fix it.

In that case **expected_stable_version** would be `v0.19.0`, right?

Reply:
yes, fixed version